enhancements/network-observability.md

```diff
@@ -0,0 +1,367 @@
+The enhancement is at [Enable flows collection](https://github.com/mariomac/enhancements/blob/enable-flows-collection/enhancements/network/enable-flows-collection.md).
+
+#### Flow Enricher
+The Flow Enricher enhances the IPFIX data with Kubernetes-related
+information.  It watches for updates on pods, services and daemonsets
+using the Kubernetes API.  It merges data from Flow Collector and
+Kubernetes to be able to display pod-to-pod traffic.  It is also
+responsible for writing out the data to Storage.  See Network Observability
+Flow Enricher enhancement for more details.
+
+#### Storage
+Storage stores the IPFIX/Kubernetes data.  It provides a REST API to
+retrieve this data.  The minimum storage requirement is 200 GB.  Because
+of the potential large amount of data, the data retention policy is 24
+hours.  This component is [Grafana Loki](https://github.com/grafana/loki).
+See Network Observability Storage enhancement for more details.
+
+
+### Operators
+Two new operators will be required for network observability.  They are
+the Network Observability Operator and the Loki Operator.
+
+#### Network Observability Operator (NetObserv)
+The Network Observability Operator will need to be installed from OperatorHub
+to enable this feature.  This operator has a dependency on the Loki
+Operator.  This is an OpenShift Console dynamic plugin that is responsible
+for defining resources and enabling collection of NetFlow similar to
+Red Hat OpenShift Logging.  This can be done from Web Console or CLI.
+
+The operator will follow the [Operator Lifecycle Manager](https://olm.operatorframework.io/)
+(OLM) model which is a component of the [Operator Framework](https://github.com/operator-framework).
+This allows Network Observability to release on its own cycle, although the
+plan is to follow the OCP's release dates and leverage this benefit
+for hot fixes only.
+
+***Side note:***<br>
+Consideration was evaluated in terms of *not* introducing a new Network
+Observability Operator.  It didn't make sense to combine with Cluster
+Network Operator or Cluster Monitoring Operator since network observability
+is an opt-in feature that requires resources.  Combining with Red Hat
+OpenShift Logging Operator (formerly Cluster Logging Operator) will limit
+the ability to enable one and not the other.
+
+#### Loki Operator
+The Loki Operator is a separate project at
+https://github.com/ViaQ/loki-operator but is required for network
+observability.  It manages Grafana Loki, which is the component that will be
+used to store NetFlows.  It will be installed in its own namespace with the
+intention that if another component wants to use Loki, it should create its
+own instance.
+
+***Side note:***<br>
+This is a move away from Elasticsearch due to licensing restrictions.
+Open Distro for Elasticsearch was considered, but Loki was favored due to other
+components that plan to ultimately use Loki.
```

Comment:
Who is going to own loki and the loki operator and ensure those get shipped and supported appropriately?